scripts/word_embeddings/evaluate_fasttext_bin.py

```diff
@@ -0,0 +1,177 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Loading and evaluation of pretrained fasttext word embeddings
+================================================================
+
+This example shows how to load the binary format containing word and subword
+information and perform intrinsic evaluation of word embeddings trained with
+the facebookresearch/fasttext implementation using a variety of datasets all
+part of the Gluon NLP Toolkit.
+
+"""
+
+import argparse
+import logging
+import os
+import struct
+
+import mxnet as mx
+import numpy as np
+
+import evaluation
+import gensim
```

Comment:
We may need to tell the users that they are required to install gensim to run this script.